packages/trail-fastify-plugin/lib/api.js

```diff
@@ -0,0 +1,115 @@
+const { get } = require('lodash')
```

Comment:
use strict is missing.

What's this file purpose, I'm a bit lost.

Reply:
The general project structure, including this file, is copied from the trail-hapi-plugin code. The function of the file is to support generation of the swagger spec from the api route definitions. Note that its functions are also called by the corresponding server module.